_data/icons/edexa.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV",
+    "width": 400,
+    "height": 400,
+    "format": "png"
```

Comment:
identify QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV         14:41:38
QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV JPEG 1028x1042 1028x1042+0+0 8-bit sRGB 53936B 0.000u 0:00.000

Reply:
added

Reply:
> identify QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV 14:41:38
> QmWFMLXD87rPB6KxjooTUc1a6o3P856fT8WKUSA6VhToHV JPEG 1028x1042 1028x1042+0+0 8-bit sRGB 53936B 0.000u 0:00.000

it should be in jpg format and size 1028x1042 ..?!

Reply:
it should be in jpg format and size 1028x1042 ..?! @ligi

Reply:
the format just has to match the metadata

Reply:
i have added the icon variable to the eip155-1995.json file and its all matching  and the format is PNG as it is as well as size are same

Reply:
not sure what is the error now is there anything need to be changed..?! @ligi

Reply:
Hi @ligi  it is actually a png image and in json it also mentioned as png file still gives this error 

![image](https://user-images.githubusercontent.com/115977226/209322922-d15f0445-20c1-4f2f-816f-c76137d5bade.png)

Reply:
@ligi  both jpg and png gives vise versa errors kindly look on to this 

![image](https://user-images.githubusercontent.com/115977226/209324378-3d62412e-ea68-414f-81d2-30e414cce7be.png)

Reply:
updated as png @ligi

Reply:
Hi @ligi  which link will it get reflected i'm not able to see the chain in website